Sources/xcproj/XCScheme.swift

```diff
@@ -169,6 +169,56 @@ final public class XCScheme {
         }
     }
 
+    final public class EnvironmentVariables {
+        public struct EnvironmentVariable {
+            public let key: String
```

Comment:
I thought about that and finally decided against it to be consistent with `xcproj` naming. 

However, since you mention that too I will refactor it and see how it goes 👍